login-workflow/example/package.json

```diff
@@ -5,11 +5,11 @@
     "dependencies": {
         "@material-ui/core": "^4.11.4",
         "@material-ui/icons": "^4.11.2",
-        "@pxblue/colors": "^3.0.1",
-        "@pxblue/icons-mui": "^2.3.0",
-        "@pxblue/react-auth-workflow": "^2.1.0",
-        "@pxblue/react-components": "^5.2.0",
-        "@pxblue/react-themes": "^6.0.0",
+        "@brightlayer-ui/colors": "^3.0.1",
+        "@brightlayer-ui/icons-mui": "^2.3.0",
+        "@brightlayer-ui/react-auth-workflow": "^2.4.0-beta.0",
```

Comment:
This dep is not required.

Reply:
This dep needs to be removed:

`"@brightlayer-ui/react-auth-workflow": "^2.4.0-beta.0",`

Our circle ci builds do not test the demo project & it should not require a manual copy/pasting of the dist folder to get this to run.

Reply:
Removed please check.